sprints/dashboard/libs/mattermost.py

```diff
@@ -0,0 +1,67 @@
+from contextlib import contextmanager
+from typing import Iterator
+from mattermostdriver import Driver
+from django.conf import settings
+
+
+class Mattermost:
+    """Class that is responsible to connect to mattermost bot."""
+
+    def __init__(self, username: str, access_token: str) -> None:
+        """
+        :param username: Username of the mattermost bot.
+        :type username: str
+        :param access_token: Access token for the bot.
+        :type access_token: str
+        """
+        connection_parameter = {
+            'url': settings.MATTERMOST_SERVER,
+            'port': settings.MATTERMOST_PORT,
+            'username': username,
+            'token': access_token
```

Comment:
@farhaanbukhsh, ok, my last comment and I'm going to sleep :P
Could we use the `settings.MATTERMOST_LOGIN_ID, settings.MATTERMOST_ACCESS_TOKEN` here and make this class a context manager itself (by implementing `__enter__` and `__exit__`) instead of adding `connect_to_mattermost`? I know that you took it from other libs, but I think that in this case it might be a better approach, as we're creating a new class that encapsulates the Mattermost driver. What do you think?

Reply:
Actually I did think about it :) and the reason I went with the current approach was because I didn't want to break the pattern. For example all lib files are following a pattern so I thought why not keep it consistent.

Reply:
and I am happy to do this by implementing `__enter__` and `__exit__` 👍

Reply:
also I don't want to use `settings.MATTERMOST_LOGIN_ID, settings.MATTERMOST_ACCESS_TOKEN` because what if we will use multiple bots?

Reply:
> what if we will use multiple bots?

Let's have only one bot for now; finding cool names for more of them is going to take a lot of time ;)
Jokes aside, the current code would need some changes to support more of them anyway, e.g. specifying different `username:token` pairs in the configurations, a way to pass the correct pair to the context manager, etc. So let's not worry about this, as we don't need it now.

Reply:
sure thing 👍